### PR TITLE
epic-85: deploys sidecar epic into dev

### PR DESCRIPTION
**Pull request by mgibson-casper**
### 😤 Problem / Goals
-

### 🤓 Solution
-

### 🗒️ Additional Notes
-

### CODE REVIEW

File: app/src/accounts/accounts.controller.ts [hunk 1]
```diff
@@ -8,15 +8,15 @@ import {
 
 import { AccountsService } from "./accounts.service";
 
-export class AccountByHashOrPublicKeyParamDtp {
+export class AccountByHashOrPublicKeyParamDto {
```
Comment (DorianKwan) [inline app/src/accounts/accounts.controller.ts:11]: What is Dtp vs Dto?

File: app/src/types/deploy.ts [hunk 2]
```diff
@@ -29,7 +29,7 @@ export interface JsonDeployWasmSession {
 }
 
 export interface GetDeploy {
-  timestamp: number;
+  timestamp: number | string;
```
Comment (DorianKwan) [inline app/src/types/deploy.ts]: Should this just be a `string`?

File: app/src/accounts/accounts.controller.ts (hunk 1 above)
Comment (mgibson-casper) [reply app/src/accounts/accounts.controller.ts:11]: It was initially a typo. It stands for 'data transfer object'. Essentially what Nestjs calls anything defined to be crossing the wire from client to server

File: app/src/types/deploy.ts (hunk 2 above)
Comment (mgibson-casper) [reply app/src/types/deploy.ts]: Yup yup, it's been updated!

**Review by Thistle-Strum — APPROVED** [review]

File: app/src/accounts/accounts.controller.ts (hunk 1 above)
Comment (DorianKwan) [reply app/src/accounts/accounts.controller.ts:11]: Awesome!

File: app/src/types/deploy.ts (hunk 2 above)
Comment (DorianKwan) [reply app/src/types/deploy.ts]: Perfect!

**Review by DorianKwan — APPROVED** [review]
LGTM!

**Review by valfaro123 — APPROVED** [review]

**Pull request merged**